clean_project/search.py

```diff
@@ -0,0 +1,51 @@
+"""
+Search a word in a file and return all the occurrence specified the line numbers
+"""
+import argparse
+import json
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+def search(search_word, json_path):
+    """
+    Print all the occurrence of a given word
+    :param search_word: Word given to search
+    :param json_path: Indexing file
+    :return: List of files and numbers of file when search_word occurrence
+    """
+    with open(json_path) as json_file:
```

Comment:
search is much better now, but you're opening the index for every search, which is a bit inconvenient. you should probably make this loading happen only once, instead of every search. any idea?